locale/Makefile

```diff
@@ -7,60 +7,7 @@
 # make clean - clean everything
 #
 DOMAIN = hammer-cli
-VERSION = $(shell ruby -e 'require "rubygems";spec = Gem::Specification::load("../hammer_cli.gemspec");puts spec.version')
-POTFILE = $(DOMAIN).pot
-MOFILE = $(DOMAIN).mo
-POFILES = $(shell find . -name '$(DOMAIN).po')
-MOFILES = $(patsubst %.po,%.mo,$(POFILES))
-POXFILES = $(patsubst %.po,%.pox,$(POFILES))
-EDITFILES = $(patsubst %.po,%.edit.po,$(POFILES))
-TIMESTAMPFILES = $(patsubst %.po,%.po.time_stamp,$(POFILES))
+VERSION = $(shell ruby -e 'require "rubygems"; spec = Gem::Specification::load("../hammer_cli.gemspec"); puts spec.version')
+MAIN_MAKEFILE = $(shell ruby -e 'require "hammer_cli"; puts HammerCLI::I18n.main_makefile')
```

Comment:
@tstrachota I found out we use `bundle exec` already in the `extract-strings` target so I think it makes sense to include it here as well. It is pointless to expect calling any of the targets outside of devel env anyway, isn't it?